dolphinscheduler-python/src/main/java/org/apache/dolphinscheduler/server/PythonGatewayServer.java

```diff
@@ -177,7 +177,7 @@ public Long createOrUpdateProcessDefinition(String userName,
         long projectCode = project.getCode();
         Map<String, Object> verifyProcessDefinitionExists = processDefinitionService.verifyProcessDefinitionName(user, projectCode, name);
 
-        if (verifyProcessDefinitionExists.get(Constants.STATUS) != Status.SUCCESS) {
+        if (verifyProcessDefinitionExists.get(Constants.STATUS) == Status.PROCESS_DEFINITION_NAME_EXIST) {
```

Comment:
I see the code, it seems the status may also be `PROJECT_NOT_FOUNT` or `USER_NO_OPERATION_PROJECT_PERM` here?

Reply:
Hi wenjun, we would create project with assigned user before code running to this line. So I think we nearly zero percent to miss this situation. Until other process drop project or user object after I create it just now.

Reply:
OK, I got you. 
I suggest adding a log if the status is neither SUCCESS and PROCESS_DEFINITION_NAME_EXIST.
This change is not mandatory, It's up to you.

Reply:
Good suggestions, I would add it to codebase, thanks